ksqldb-common/src/main/java/io/confluent/ksql/metrics/ConsumerCollector.java

```diff
@@ -69,17 +68,26 @@ public void configure(final Map<String, ?> map) {
     if (id == null) {
       id = (String) map.get(ConsumerConfig.CLIENT_ID_CONFIG);
     }
+
+    final KsqlConfig config = new KsqlConfig(map);
```

Comment:
Do we have to first wrap it in a `KsqlConfig` and then get by the config name? I saw that it's default value is `""` anyways so seems just getting from the `map` with static `parseStringAsMap` is fine too.